src/styles/blocks/_card.scss

```diff
@@ -0,0 +1,85 @@
+.card {
+  width: $card-width;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 4px;
+  padding: 32px 16px 16px;
+
+  &__image {
+    align-self: center;
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__description {
+    @include main-text;
+    line-height: 18px;
+    color: $main-color-accent;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    @include extra-small-text;
+    line-height: 14px;
+    color: $secondary-color;
+    margin-bottom: 16px;
+  }
+
+  &__review,
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
```

Comment:
I guess its better to use mixin for solutions like this 'coz you wrote same selectors twice)